dev/NavigationView/NavigationViewItemsFactory.h

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "common.h"
+#include "ElementFactory.h"
+
+class NavigationViewItemsFactory :
+    public winrt::implements<NavigationViewItemsFactory, ElementFactory>
+{
+public:
+    NavigationViewItemsFactory();
+
+    void UserElementFactory(winrt::Windows::UI::Xaml::IElementFactory elementFactory) { m_userElementFactory = elementFactory; };
```

Comment:
>Windows [](start = 35, length = 7)

Why is this specifically a windows type? doesn't IElementFactory exist in MUX as well?